tests/Feature/Api/Validation/AddressTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Feature\Api\Validation;
+
+use GuzzleHttp\Exception\ClientException;
+use jsamhall\ShipEngine\Address\Address;
+use PHPUnit\Framework\TestCase;
+use Tests\Mocks\Validation\AddressMock;
+
+class AddressTest extends TestCase
+{
+
+    public function testValidateAddress()
+    {
+        //Arrange
```

Comment:
For all these Arrange/ActAssert, can you put 1 space?

so like
```
// Arrange
// Act
// Assert
```

Reply:
AAA group given the space they need.